server/assets/marketing_next/js/hooks.js

```diff
@@ -0,0 +1,66 @@
+const Hooks = {
+  NavbarDropdown: {
```

Comment:
This is basically a special variant of our dropdown in Noora. I'd maybe consider reusing or replicating the `Dropdown.js` hook and having a very similar structure. I don't see a strong need to have a completely different way of implementing this specific dropdown. Plus we lose some small niceties like closing the dropdown on escape.